packages/runtime/garbage-collector/src/utils.ts

```diff
@@ -201,19 +228,12 @@ export class GCDataBuilder implements IGarbageCollectionData {
      */
     public prefixAndAddNodes(prefixId: string, gcNodes: { [ id: string ]: string[]; }) {
         for (const [id, outboundRoutes] of Object.entries(gcNodes)) {
-            let normalizedId = id;
-            // Remove any starting slashes from the id.
-            while (normalizedId.startsWith("/")) {
-                normalizedId = normalizedId.substr(1);
-            }
-
+            // Remove any leading slashes from the id.
+            let normalizedId = trimLeadingSlashes(id);
```

Comment:
Might it be better to both trim leading and trailing slashes here? because of what you do below

Reply:
That's what I did at first but the problem is when the id is `/`. If we do both here and then add the prefix, we end up with a `/` at the ends. We could add special handling for `/` or add another step to trim the last slash in this case. I thought this was the cleanest way to do it.

Reply:
We should comment this edge case

Reply:
Will add a comment for this.